src/style.css

```diff
@@ -0,0 +1,72 @@
+
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+form:first-of-type {
+  margin: 20px 0;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar {
+  height: 70px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir-Heavy, sans-serif;
+  color: #3d4e61;
+}
+
+.search-bar::placeholder {
+  font-family: Avenir-Book, sans-serif;
+  color: #3d4e61;
+}
+
+.search-s {
+  height: 42px;
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+.search-l {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.block {
+  position: relative;
+}
+
+.img-l {
+  position: absolute;
+  left: 19px;
+  top: 19px;
+}
+
+.img-s {
```

Comment:
in the small search-bar image is a bit smaller. You can specify width for this image